pinot-core/src/main/java/org/apache/pinot/core/query/selection/SelectionOperatorUtils.java

```diff
@@ -289,6 +289,7 @@ public static DataTable getDataTableFromRows(Collection<Object[]> rows, DataSche
             dataTableBuilder.setColumn(i, (int[]) columnValue);
             break;
           case LONG_ARRAY:
+          case TIMESTAMP_ARRAY:
```

Comment:
We should fix `DataSchema.ColumnDataType.getStoredType()` to include `BOOLEAN_ARRAY` and `TIMESTAMP_ARRAY` instead of adding the type here

Reply:
thanks, will do that